librz/util/file.c

```diff
@@ -358,6 +358,19 @@ RZ_API char *rz_file_path(const char *bin) {
 	return strdup(bin);
 }
 
+RZ_API char *rz_file_binsh(void) {
+	char *bin_sh = rz_sys_getenv("SHELL");
+	if (RZ_STR_ISEMPTY(bin_sh)) {
+		free(bin_sh);
+		bin_sh = rz_file_path("sh");
+		if (RZ_STR_ISEMPTY(bin_sh)) {
+			free(bin_sh);
+			bin_sh = strdup("/bin/sh");
+		}
+	}
+	return bin_sh;
```

Comment:
```suggestion
	char *bin_sh = rz_sys_getenv("SHELL");
	if (!RZ_STR_ISEMPTY(bin_sh)) {
		return bin_sh;
	}

	free(bin_sh);
	bin_sh = rz_file_path("sh");
	if (!RZ_STR_ISEMPTY(bin_sh)) {
		return bin_sh;
	}

	free(bin_sh);
	bin_sh = strdup("/bin/sh");
	return bin_sh;
```
otherwise as we add more possibilities we just end up with too much indentation.

Reply:
Thanks, applied and amended the commit.